### PR TITLE
Windows11 shutdown fix

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

**Comment by miczyg1** [comment]
Fixes: https://github.com/Dasharo/dasharo-issues/issues/562

**Comment by macpijan** [comment]
I rebased to enable CI build

**Comment by mkopec** [comment]
@miczyg1 could you summarize here what are the effects of switching to SCM on Linux?

**Comment by miczyg1** [comment]
> @miczyg1 could you summarize here what are the effects of switching to SCM on Linux?

Dock is still working on ADL as intended
